### DISCUSSION TITLE
budibase down?

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#4915] [Help]
Hello,

I use the live editor as a development environment and it looks to be down...

Just to let you know.

**Comment by dominikjannis** [comment]
I can confirm, i was active while it first degraded and then went down. Maybe this helps track down the issue:
1. Responses went increasingly slower
2. A global banner informing "Some systems are experiencing issues" or the like
4. Error messages about an invalid json response because they started with "<" (possibly getting an unexpected html response?)
5. 504
6. 503

**Comment by PClmnt** [comment]
Hi, thanks for the report! We're aware and working on it right now.
